ark/utils/spatial_lda_utils.py

```diff
@@ -0,0 +1,63 @@
+import os
+import numpy as np
+import pandas as pd
+
+def check_format_cell_table_args(cell_table, markers, clusters, fovs):
+
+    """
+    Checks the input arguments of the format_cell_table() function.
+
+    Args:
+        cell_table (pd.DataFrame):
+            A pandas DataFrame containing the columns of cell marker frequencies and/or cluster ids.
+        markers (list, str):
+            A list of strings corresponding to marker names.
+        clusters (list, int):
+            A list of integers corresponding to cluster ids.
+        fovs (str, int):
+            One of either "all_fovs" indicating all field of views are to be kept, or a list of integers corresponding
+            to the index of each field of view which should be kept.
+
+    Returns:
+        None
+    """
+
+    base_cols = [
+        "point",
+        "label",
+        "cell_size",
+        "centroid-0",
+        "centroid-1",
+        "pixelfreq_hclust_cap",
+        "name"
+    ]
+
+    # Check cell table
+    if not isinstance(cell_table, pd.DataFrame):
```

Comment:
One important part of the testing philosophy is figuring out which functions can trust each other, and which cannot. Because we'll be loading the .csv ourselves in a previous function, I think we can assume that if the user gets this far it was succesfully converted to a pandas file